frontend/catalyst/jit.py

```diff
@@ -662,7 +662,7 @@ def fn_with_transform_named_sequence(*args, **kwargs):
             results with a transform_named_sequence primitive at the beginning of the
             jaxpr. It is never executed or used anywhere, except being traced here.
             """
-            _inject_transform_named_sequence()
+            # _inject_transform_named_sequence()
             return self.user_function(*args, **kwargs)
```

Comment:
I guess this entire `fn_with_transform_named_sequence` function does not need to exist anymore now that we inject the transform sequence op somewhere else?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/1323/commits/82b494d3fff8dab373bbbb4c264e06f7bcd6a74d